components/prism-crystal.js

```diff
@@ -0,0 +1,54 @@
+(function(Prism) {
+	Prism.languages.crystal = Prism.languages.extend('ruby', {
+		number: /\b(?:0b[01_]*[01]|0o[0-7_]*[0-7]|0x[0-9a-fA-F_]*[0-9a-fA-F]|(?:[0-9][0-9_]*[0-9]|[0-9])(?:\.[0-9_]*[0-9])?(?:[eE][+-]?[0-9_]*[0-9])?)(?:_[uif](?:8|16|32|64|))?\b/,
+	});
+
+	Prism.languages.insertBefore('crystal', 'operator', {
+		keyword: [
+			/\b(?:abstract|alias|as|asm|begin|break|case|class|def|do|else|elsif|end|ensure|enum|extend|for|fun|if|ifdef|include|instance_sizeof|lib|macro|module|next|of|out|pointerof|private|protected|rescue|return|require|self|sizeof|struct|super|then|type|typeof|undef|union|unless|until|when|while|with|yield|__DIR__|__FILE__|__LINE__)\b/,
+			{
+				pattern: /(\.\s*)(is_a|responds_to)\?/,
```

Comment:
Because you took care of making all others groups non-capturing where not needed, may I suggest you add `?:` to the second group here too?